bin/pre-commit.js

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env node
```

Comment:
Will this work if you don't have node installed globally?

Reply:
Probably not, but neither will anything else in our build process.

Reply:
if it is for ci environment and not part of mvn command cycle, is ok as we use that elsewhere... 

but I guess this does require devs to have npm/node installed (not unreasonable for development, as long as it isn't required for mvn commands right @kzantow?)

Reply:
For now it's client-side only, so it only needs to run on dev machines.